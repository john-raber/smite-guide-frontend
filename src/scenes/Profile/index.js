import React from "react";
import { Container, Row, Col, Jumbotron } from "reactstrap";

import "./Profile.css";

const Profile = () => {
  return (
    <Container>
      <Row>
        <Col md={{ size: 6 }}>
          <Jumbotron>
            <h4 className="display-4">Guides</h4>
          </Jumbotron>
        </Col>
        <Col md={{ size: 6 }}>
          <Jumbotron>
            <h4 className="display-4">Trainers</h4>
          </Jumbotron>
        </Col>
      </Row>
    </Container>
  );
};

export default Profile;
