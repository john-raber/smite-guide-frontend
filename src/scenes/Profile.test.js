import React from "react";
import { shallow } from "enzyme";

import Profile from "./Profile/index";

describe("Profile", () => {
  it("renders correctly", () => {
    let wrapper = shallow(<Profile />);

    expect(wrapper.find(".Profile").length).toEqual(1);
  });

  it("matches its snapshot", () => {
    let wrapper = shallow(<Profile />);

    expect(wrapper).toMatchSnapshot();
  });
});
