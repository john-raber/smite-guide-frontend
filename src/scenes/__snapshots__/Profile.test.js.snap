// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile matches its snapshot 1`] = `
<Container
  className="Profile"
  tag="div"
>
  <Row
    tag="div"
  >
    <Col
      md={
        Object {
          "size": 6,
        }
      }
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
    >
      <Jumbotron
        tag="div"
      >
        <h4
          className="display-4"
        >
          Guides
        </h4>
      </Jumbotron>
    </Col>
    <Col
      md={
        Object {
          "size": 6,
        }
      }
      tag="div"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
    >
      <Jumbotron
        tag="div"
      >
        <h4
          className="display-4"
        >
          Trainers
        </h4>
      </Jumbotron>
    </Col>
  </Row>
</Container>
`;
